sdk/cosmos/azure-cosmos/azure/cosmos/aio/scripts.py

```diff
@@ -143,21 +150,22 @@ async def create_stored_procedure(self, body, **kwargs):
     @distributed_trace_async
     async def replace_stored_procedure(
         self,
-        sproc,
-        body,
-        **kwargs
-    ):
-        # type: (Union[str, Dict[str, Any]], Dict[str, Any], Any) -> Dict[str, Any]
+        sproc: Union[str, Dict[str, Any]],
```

Comment:
Oh wow. I'm assuming this name is intuitive to stored procedure users?

Reply:
yeah, it's also the naming present in the sync client